core/tests/webdriverio/learnerFlow.js

```diff
@@ -435,11 +442,14 @@ describe('Learner dashboard functionality', function () {
     await libraryPage.findCollection('Test Collection');
     await libraryPage.playCollection('Test Collection');
     await waitFor.pageToFullyLoad();
+
+    await waitFor.visibilityOf(
+      $(collectionExplorationSelector),
+      'Collection exploration takes too long to appear'
+    );
     // The collection player has two sets of SVGs -- one which is
     // rendered for desktop and the other which is rendered for mobile.
-    var firstExploration = browser.isMobile
-      ? await $$('.e2e-test-mobile-collection-exploration')[0]
-      : await $$('.e2e-test-collection-exploration')[0];
+    var firstExploration = await $$(collectionExplorationSelector)[0];
```

Comment:
Is it possible to move this somewhere in the utils file as a reusable function? (something around openFirstExplorationInCollection)

Reply:
Where should i move this? There is no CollectionPlayer module to interact with the collection player page.

Reply:
Feel free to create a new utils file and use it!